source/transactions-convenient-api/tests/README.md

```diff
@@ -41,8 +41,14 @@ If possible, drivers should implement these tests without requiring the test run
 the retry timeout. This might be done by internally modifying the timeout value used by `withTransaction` with some
 private API or using a mock timer.
 
+### Retry Backoff is Enforced
+
+Drivers should test that retries within `withTransaction` do not occur immediately. Ideally, set BACKOFF_INITIAL 500ms
+and configure a failpoint that forces one retry. Ensure that the operation took more than 500ms so succeed.
```

Comment:
This test would not work as described because jitter is non-deterministic. 

An alternative test would be to use a failpoint to fail the transaction X times and then assert the overall time is larger than some threshold.